SonarQube.TeamBuild.PreProcessor/Resources.resx

```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="AssemblyDescription" xml:space="preserve">
-    <value>MSBuild SonarQube Runner Pre-processor</value>
+    <value>SonarQube Scanner for MSBuild Pre-processor</value>
```

Comment:
@bgavrilMS cc @duncanpMS  - IMO `Pre-processor` is a highly technical term that won't speak to users.. why not replace it by "Begin step" ?